pkg/manager/member/utils.go

```diff
@@ -496,6 +496,29 @@ func CreateOrUpdateService(serviceLister corelisters.ServiceLister, serviceContr
 	return nil
 }
 
+// CreateOrUpdateConfigMap check configmap before create or update the new configmap
+func CreateOrUpdateConfigMap(configMapLister corelisters.ConfigMapLister, controller controller.ConfigMapControlInterface, newCm *corev1.ConfigMap, obj runtime.Object) (*corev1.ConfigMap, error) {
+	oldCm, err := configMapLister.ConfigMaps(newCm.Namespace).Get(newCm.Name)
+	if err != nil && !errors.IsNotFound(err) {
+		return nil, fmt.Errorf("CreateOrUpdateConfigMap error: failed to get configMap %s/%s, error: %s", newCm.Namespace, newCm.Name, err)
+	}
+	if errors.IsNotFound(err) {
+		cm, err := controller.CreateConfigMap(obj, newCm)
+		if err != nil && !errors.IsAlreadyExists(err) {
+			return nil, err
+		}
+		return cm, nil
```

Comment:
Yes, same to the `func (c *realGenericControlInterface) CreateOrUpdate`.

Reply:
There is no such issue in `func (c *realGenericControlInterface) CreateOrUpdate` after checking the code?
I think you can just return error here is it's `errors.IsAlreadyExists`

Reply:
It'll also return error when `func (c *realGenericControlInterface) CreateOrUpdate` create configmap error.  
But can't return error directly when meets `errors.IsAlreadyExists` cause we should check if we need to update the old configmap.

Reply:
I mean it will not lose the update when the cache is not synced since it `Create()` first and update if exist.
But here, there is a chance it will lose the update.

Reply:
But if we want to update the configmap accurately everytime, we need to call API to the kube-apiserver directly during each sync, which is what this PR tries to fix😂.
Maybe it takes little effect to sync configmap in this sync turn or next sync turn when cache not syned?

Reply:
just change L507
from 
`if err != nil && !errors.IsAlreadyExists(err) {`
to 
`if err != nil  {`

this seems hardly happen and it's still safe to return error and let it re-sync?

Reply:
Got it, I changed the check logic to make the `UpdateConfigMap` work when `errors.IsAlreadyExists` occurs.